app/services/idv/vendor_validated.rb

```diff
@@ -1,21 +0,0 @@
-module Idv
-  module VendorValidated
-    extend ActiveSupport::Concern
-
-    def vendor_validate
```

Comment:
I agree with that last article. I should have mentioned that I there is a follow up PR that will replace the parent Step class with a Module because the only purpose of this Step class is to hold code in common with the other classes. 

The first test I use to determine whether to use a [mixin](https://en.wikipedia.org/wiki/Mixin) or inheritance is if the parent class will ever be instantiated. In this case, Step is not. There is no abstract Step concept in my mind. A step is really just a form submission. In this case, we have 3 forms that share certain methods, which is a great fit for a mixin: "a class that contains methods for use by other classes without having to be the parent class of those other classes."

Here is another article I like on this subject: http://naildrivin5.com/blog/2012/12/19/re-use-in-oo-inheritance.html